core/src/main/java/com/scalar/db/sql/BindMarker.java

```diff
@@ -0,0 +1,12 @@
+package com.scalar.db.sql;
+
+public interface BindMarker extends Term {
+
+  static BindMarker of() {
+    return PositionalBindMarker.INSTANCE;
+  }
+
+  static BindMarker of(String name) {
+    return new NamedBindMarker(name);
+  }
```

Comment:
Honestly, I'm not sure this method name (`of()`) is good.

Reply:
A more descriptive alternative could be : 
```
BindMarker.positional() 
BindMarker.named(String name)
```

Reply:
Thank you! They are better names. I will change the methods names.